classes/core/APIRouter.inc.php

```diff
@@ -96,8 +96,30 @@ function route($request) {
 
 		$sourceFile = sprintf('api/%s/%s/index.php', $this->getVersion(), $this->getEntity());
 
+		$dispatcher = $this->getDispatcher();
+		$dispatcher->setAuthorizationDenialCustomHandler(function($authorizationErrorCode) {
+			$msg = '';
```

Comment:
Since there's a `default:` clause, I think this is dead code.

Reply:
I believe If the variable does not get created before the switch statement, PHP will trigger a `Notice` under strict mode because the variable is being referenced from an outer scope (line 113).